ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -62,7 +62,14 @@ const AGREE_CHECKBOX = {
   validate: validators.required,
 }
 
-const FORM_FIELDS = [PROJECT_DESC_FIELD, UPLOAD_PEDIGREE_FIELD, GENOME_VERSION_FIELD, AGREE_CHECKBOX]
+const DATA_BUCK_FIELD = {
+  name: 'dataPath',
+  label: 'Google Path to the VCF files',
```

Comment:
This should be "Path to the Joint Called VCF". I also think adding a `labelHelp` here would be helpful, as that will render some additional help text on hover next to the label. I think it should say something like "File path for a joint called VCF available in the workspace files. If The VCF is split, provide the path to the directory containing the split VCF"